bazel/http-server/main.ts

```diff
@@ -15,7 +15,11 @@ const {rootPaths, historyApiFallback, enableDevUi, environmentVariables, port, r
   process.argv.slice(2),
 )
   .strict()
-  .option('port', {type: 'number', default: 4200})
+  .option('port', {
+    type: 'number',
+    default: process.env['PORT'] !== undefined ? Number(process.env['PORT']) : 4200,
```

Comment:
Do we want to do a `NaN` check as well?  Probably not strictly necessary, but figured I would ask.

Reply:
I did consider it, but a runtime error is okay if this is off IMO; so will merge for nkw